nautobot/core/context_processors.py

```diff
@@ -34,9 +35,22 @@ def settings(request):
     Expose Django settings in the template context. Example: {{ settings.DEBUG }}
     """
 
+    try:
+        view_class = request.resolver_match.func.view_class
+    except AttributeError:
+        # Use this method to import the view class as apps (like example_plugin)
+        # do not have the 'view_class' attribute.
```

Comment:
Can you elaborate on this? I don't understand offhand why plugin apps would behave differently from core apps in this regard.